README.md

```diff
@@ -1,75 +1,166 @@
-# full-stack-test-template
-
-A monorepo template to use for our full-stack test.
+# Percipio Take Home Challenge
 
 ## Expectations
 
-As the name implies this is meant to be a full-stack test, but no full-stack engineer
-focuses equally on frontend and backend. Please complete both ends of the test, but
-feel free to lean more into the area you are most passionate about, and let us know 
-in this README which side you're leaning.
+Please make note of the following expectations before you begin coding:
 
-There is no time limit imposed by us or required response time.
-This is your opportunity to stand out and show us your skills.
-We're excited to see what you come up with!
+1. **This challenge consists of two pieces - (1) the base challenge, and (2) is the enhancement**. We expect you to complete the base challenge and only one of the enhancements from the list further down this readme.
+2. **We will only assess you for the code you submit that completes the base challenge and the one enhancement**. Please indicate in your submission which enhancement you completed.
+3. **We expect you to spend 2-4 hours on this challenge**. If you’re unable to complete all the requirements in that time frame, feel free to explain in your write-up any of the thoughts or plans you would have completed with more time.
+4. **We do not expect a full scale application with working navigation, complex animations, etc. You should only develop what you need in order to accomplish the goal we've given.**
+5. The purpose of this challenge is to get a sense of your work, and to serve as the focal point of your technical interview. Expect the code you submit to come up in your technical interview.
 
-## Process
+## The Process
 
-1. Start by using this template. You can do this by clicking the "Use this template" button in the top right, then "Create a new reposistory". You should make this new repo private.
-2. Read the details about the test below and complete the test to your liking.
-3. Invite the code review team to your repository and email us to let us know it's ready. If the list of GitHub users has not already been provided to you, work with your contact here to get that list.
-4. We will review your work and schedule a technical interview where you will meet some of our engineers and discuss this test.
+Before you start coding, make sure you have the following installed on your machine:
 
-# The Test
+1. `nvm` or Node.js
+2. `pnpm` (we recommend installing with Corepack)
+3. (Optional) nx for running workspaces
 
-There are two sections to the test: API and UI.
-There are READMEs in each workspace, though they are just the default READMEs generated by NestJS & Vite.
-You can use them for reference or modify/replace them with anything you want to document.
+When you are ready to start coding:
 
-## API
+1. Clone this repo
```

Comment:
ooh just realized this is a template repo, so we can pull over the existing instructions from our fullstack test:
```suggestion
1. Start by using this template. You can do so by clicking the "Use this template" button in the top right of the GitHub UI, then "Create a new repository". Make this new repo private.
```

Reply:
ooh actually looking at those docs, I think most of that `Process` section would be good to include... (although we may want to run this by Eddie to ensure he's expecting this same flow)
```
* Start by using this template. You can do so by clicking the "Use this template" button in the top right of the GitHub UI, then "Create a new repository". Make this new repo private.
* Read the details about the test below and complete the test.
* Invite the code review team to your repository and email us to let us know it's ready. If the list of GitHub users has not already been provided to you, work with your contact here to get that list.
* We will review your work and schedule a technical interview where you will meet some of our engineers and discuss this test.